src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2195,7 +2195,7 @@
     <Field Name="InitialBindingReceiverIsSubjectToCloning" Type="ThreeState"/>
     <Field Name="PropertySymbol" Type="PropertySymbol"/>
     <!-- The property accessor kind, set for properties with synthesized backing field only. -->
```

Comment:
is it also the case that this will only have a non-Unknown value when auto accessors are being used?

Reply:
The value will be set to something other than `AccessorKind.Unknown` for classic auto properties and for properties that use `field`. Properties that use `field` may include an auto-implemented accessor.